src/Tizen.NUI/src/public/WebView/WebViewPageLoadEventArgs.cs

```diff
@@ -45,10 +44,11 @@ public WebView WebView
         }
 
         private string _pageUrl;
+
         /// <summary>
         /// The url string of current webpage.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 8 </since_tizen>
         public string PageUrl
```

Comment:
I think this will not be necessary : WebViewPageLoadEventArgs.PageUrl

User may get the page url from the WebView.Url

Reply:
removed.